doc/source/dev/index.rst

```diff
@@ -19,6 +19,21 @@ we list them in alphabetical order):
 - Website design and development
 - Writing technical documentation
 
+We understand that everyone has a different level of experience,
+also NumPy is a pretty well-established project, so it's hard to
+make assumptions about an ideal "first-time-contributor". 
+So, that's why we don't mark issues with the "good-first-issue"
+label. Instead, you'll find [issues labeled "Sprintable."](https://github.com/numpy/numpy/labels/sprintable).
+These issues can either be:
+
+- **Easily fixed** when you have guidance from an experienced
+contributor (perfect for working in a sprint).
+- **A learning opportunity** for those ready to dive deeper,
+even if you're not in a sprint. 
```

Comment:
I think indention here will fix the CI failure.
```suggestion
- **Easily fixed** when you have guidance from an experienced
  contributor (perfect for working in a sprint).
- **A learning opportunity** for those ready to dive deeper,
  even if you're not in a sprint. 
```

Reply:
Shouldn't the lint test catch this? That seems more logical to me.